resolwe/permissions/mixins.py

```diff
@@ -16,6 +16,25 @@
 from .utils import check_owner_permission, check_public_permissions, check_user_permissions, update_permission
 
 
+class CurrentUserPermissionsSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """Current user permissions serializer."""
+
+    id = serializers.IntegerField()  # pylint: disable=invalid-name
+    name = serializers.CharField(max_length=100)
+    permissions = serializers.ListField(
+        child=serializers.CharField(max_length=30)
+    )
+
+    def get_fields(self):
+        """Return serializer's fields."""
+        fields = super(CurrentUserPermissionsSerializer, self).get_fields()
+
+        # Manually add `type` as it is a reserved keyword in Python.
+        fields['type'] = serializers.CharField(max_length=50)
```

Comment:
This is not true, `type` is not a reserved keyword and you can easily use it in attribute access. That it looks reserved is probably an issue with syntax highlighting.

Reply:
Correct, thanks for noticing that.